script/repour-run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+
+WORK_DIR="/home/repour"
+cd "$WORK_DIR"
+
+USR_ID=1001
+USR=$(getent passwd "$USR_ID" | cut -d: -f1)
+
+./download-pme.sh "latest"
```

Comment:
@jsenko if this is a production image and not for test/devel purpose only, it have to include executable binaries at image build time.